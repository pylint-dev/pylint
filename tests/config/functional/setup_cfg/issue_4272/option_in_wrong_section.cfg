# Test for the behavior of https://github.com/PyCQA/pylint/issues/4272
[pylint.MASTER]
load_plugins=pylint.extensions.bad_builtin
persistent=no
[pylint.DEPRECATED_BUILTINS]
bad-functions=print,map
