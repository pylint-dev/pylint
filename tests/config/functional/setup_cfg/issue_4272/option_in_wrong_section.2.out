************* Module {abspath}
{relpath}:1:0: E0015: Unrecognized option found: load_plugins, bad-functions (unrecognized-option)
